bin/setup

```diff
@@ -3,6 +3,39 @@ require 'pathname'
 require 'fileutils'
 include FileUtils
 
+def detect_os
+  if `uname` == "Darwin"
```

Comment:
I think this line will always fail because `uname` returns 'Darwin\n'.

Reply:
whoops! need to add a `chomp` to the end :)

Reply:
Done!